docker-compose.yml

```diff
@@ -1,5 +1,16 @@
 version: "3.3"
 services:
+  postgres:
+    image: postgres
```

Comment:
I think it's better to specify the postgres version of this image. Heroku at the moment uses postgres 15, so I reckon that can be the pinned version.